asmcomp/cmmgen.ml

```diff
@@ -2481,7 +2499,7 @@ let transl_function f =
              fun_args = List.map (fun id -> (id, typ_val)) f.params;
              fun_body = transl empty_env body;
              fun_fast = !Clflags.optimize_for_speed;
-             fun_dbg  = f.dbg; }
+             fun_dbg  = f.dbg}
```

Comment:
Please remove this change and the whitespace-only changes in the following thunks.